tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/Scripts.spec.ts

```diff
@@ -0,0 +1,250 @@
+import {ConstantHelper, test} from "@umbraco/playwright-testhelpers";
+import {expect} from "@playwright/test";
+
+test.describe('Script tests', () => {
+
+  const scriptName = 'TestScript';
+  const scriptPath = scriptName + '.js';
+  const scriptFolderName = 'TestScriptFolder';
+
+  test('can create a empty script', async ({page, umbracoApi, umbracoUi}) => {
+    // Arrange
+    await umbracoApi.script.ensureNameNotExists(scriptPath);
+
+    // Act
+    await page.goto(umbracoApi.baseUrl + '/umbraco');
+    await umbracoUi.goToSection(ConstantHelper.sections.settings);
+    await page.locator('umb-tree-item', {hasText: 'Scripts'}).getByLabel('Open actions menu').click({force: true});
+    await page.getByLabel('New empty script').click();
+    // TODO: Change the label to script name when the label is updated
+    await page.getByLabel('template name').fill(scriptName);
+    // TODO: Remove this timeout when frontend validation is implemented
+    await page.waitForTimeout(1000);
```

Comment:
You can use await page.waitForLoadState('networkidle') to wait for the network to idle till it loads

Reply:
I'm not totally sure if it is a good way to go 🙁 Playwright themselves discourage the use of 'networkidle'

Reply:
you're right. so just ignore this comment.